modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc

```diff
@@ -56,9 +56,19 @@ apiVersion: "k8s.cni.cncf.io/v1"
 ----
 <1> The type is `bond`.
 <2> The `ifname` attribute specifies the name of the bond interface.
-<3> The `failover` attribute is mandatory for active-backup mode.
-<4> The `linksInContainer=true` flag tells the Bond CNI that the interfaces required are to be found inside the container. By default Bond CNI looks for these interfaces on the host which does not work for integration with SRIOV/Multus.
-<5> The `links` section defines which interfaces will be used to create the bond. By default, Multus names the attached interfaces as: "net", plus a consecutive number, starting with one.
+<3> The `mode` attribute specifies the bonding mode. 
++
+[NOTE]
+====
+The bonding modes supported are:
+
+* 0 - balance-rr
+* 1 - active-backup
+* 2 - balance-xor
+====
+<4> The `failover` attribute is mandatory for active-backup mode.
+<5> The `linksInContainer=true` flag tells the Bond CNI that the interfaces required are to be found inside the container. By default Bond CNI looks for these interfaces on the host which does not work for integration with SRIOV/Multus.
```

Comment:
```suggestion
<5> The `linksInContainer=true` flag tells the Bond CNI that the interfaces required are to be found inside the container. By default Bond CNI looks for these interfaces on the host which does not work for integration with SR-IOV/Multus.
```

Reply:
This callout could do a better job of following these style guidelines.

> Do not assign human characteristics to inanimate objects, which are incapable of human behaviors and emotions. As much as possible, focus technical information on users and their actions, not on a product and its actions.

> In most cases, do not use slashes in text because they can be ambiguous.

> Write in active voice and the present tense as much as possible.